.github/workflows/pytest.yml

```diff
@@ -20,6 +20,8 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+      with:
+        ref: development
```

Comment:
The problem if you do this is that every PR onwards will checkout from development :)

Reply:
Yeah this ref development might work but if we create a parallel file? It will be duplicated code with `pytest.yml` but I cannot think of another way. 

So maybe let us only create a very short schedule.yml that just install with ref development and run pytest on 3.8 python?

Reply:
sure